cli/cook/cli.py

```diff
@@ -90,9 +92,10 @@ def run(args):
 
     log_format = '%(asctime)s [%(levelname)s] %(message)s'
     if verbose:
+        logging.getLogger('').handlers = []
```

Comment:
Without this, we are not guaranteed that our `basicConfig` call will have any effect